src/general/index.ts

```diff
@@ -110,3 +110,57 @@ export const getSelectedTabIndex = (tabs:any[], selectedTabName:string):number =
     return defaultIndex;
   }
 };
+
+
+/**
+ * This method is used to check if the given string check for characters from a-z, A-Z, 0-9.
+ *
+ * @param str - String that you want to check for the above characters
+ *
+ * @example
+ * ```
+ * if(isAlphanumericString('aaAa123')) {
+ *  return true
+ * }
+ *
+ * if(isAlphanumericString('aaAa_98-')) {
+ *  return false
+ * }
+ * ```
+ *
+ */
+export const isAlphanumericString = (str: string) => {
+  const regexForAlphaNumericString = /^[a-z0-9]+$/i;
+
+  return regexForAlphaNumericString.test(str);
+};
+
+
+
+/**
+ * This method checks for all characters to be number between 0-9 and pincode length to be 6
+ *
+ * @param {string} pincode - string entered in input element
+ *
+ * @example
+ *```
+ * if(isValidPincode('123456')) {
+ *  return true
+ * }
+ *
+ * if(isValidPincode('1234aa')) {
+ *  return false
+ * }
+ *
+ * if(isValidPincode('12345')) {
+ *  return false
+ * }
+ * ```
+ *
+ */
+export const isValidPincode = (pincode: string) => {
```

Comment:
this method should handle both number or string as people might sent number also.